File: 2639-SeparateTheDigitsInAnArray/2639-SeparateTheDigitsInAnArray.py
# Last updated: 8/13/2026, 10:10:58 PM
class Solution:
    def separateDigits(self, nums: List[int]) -> List[int]:
        res=[]
        for num in nums:
            for digit in str(num):
                res.append(int(digit))
        return res
    
        

File: 2639-SeparateTheDigitsInAnArray/2639-SeparateTheDigitsInAnArray.java
// Last updated: 8/13/2026, 10:11:05 PM
class Solution {
    public int[] separateDigits(int[] nums) {
        List<Integer> res=new ArrayList<>();
        for(int x:nums)
        {
            List<Integer> temp= new ArrayList<>();
            while(x>0)
            {
                temp.add(x%10);
                x/=10;
            }
            Collections.reverse(temp);
            res.addAll(temp);
        }
        return res.stream().mapToInt(i->i).toArray();
        
    }
}